rs/utils/src/str.rs

```diff
@@ -140,6 +143,10 @@ fn test_ellipsize() {
 
     assert_eq!("123454321".ellipsize(9, 50), "123454321");
 
+    assert_eq!("123454321".ellipsize(2, 0), "");
```

Comment:
Please add cases for 0, 1, 2 before the case of 3.

Reply:
Good point. Moved. Thanks!